tests/subsys/logging/log_custom_format/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
imo it's better to extend `log_switch_format` suite than adding new one.
Note that in `testcase.yaml` of that test you can setup a configuration with and without custom formatting and then skip the test based on presence of `CONFIG_LOG_CUSTOM_SUPPORT`)

Reply:
I started going down that route indeed then switch to a its own testcase. Will do that then